pkg/jx/cmd/install.go

```diff
@@ -1553,7 +1553,12 @@ func (options *InstallOptions) configureGitOpsMode(configStore configio.ConfigSt
 				return "", "", err
 			}
 		}
-		gitOpsDir = filepath.Join(options.Flags.Dir, "jenkins-x-dev-environment")
+
+		if options.Flags.DefaultEnvironmentPrefix == "" {
+			options.Flags.DefaultEnvironmentPrefix = strings.ToLower(randomdata.SillyName())
```

Comment:
If you search for the usage of `options.Flags.DefaultEnvironmentPrefix` you find that we assign a value to it at multiple places. I think that is confusing and the prefixes might in the end not match up (which would be nice).

Would it not be better to have a `ValidateOptions` function of the sort at the beginning of `Run` and set a valid `DefaultEnvironmentPrefix` once?

Reply:
good call! I moved it into check flags.